index.html

```diff
@@ -0,0 +1,178 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Digital Marketing Summit</title>
+
+</head>
+
+<body>
+    <header>
+        <nav>
+            <div class="navbar">
+                <div class="container nav-container">
+                    <input class="checkbox" type="checkbox" name="" id="" />
+                    <div class="hamburger-lines">
+                        <span class="line line1"></span>
+                        <span class="line line2"></span>
+                        <span class="line line3"></span>
+                    </div>
+
+                    <ul class="menu-items">
+                        <li><a href="#">Home</a></li>
+                        <li><a href="/about.html">About</a></li>
+                        <li><a href="#">Program</a></li>
+                        <li><a href="#">Sponsor</a></li>
+                        <li><a href="#">News</a></li>
+                        <li><a href="#">CC Campaign</a></li>
+                    </ul>
+                </div>
+                <div class="container desktop-nav-container">
+
+                    <a href="/index.html"><img
+                            src="https://99designs-blog.imgix.net/blog/wp-content/uploads/2020/05/attachment_62260073-e1588694361186.png?auto=format&q=60&fit=max&w=930"
+                            class="nav-logo"></a>
+
+
+
+                    <ul class="desktop-menu-items">
+                        <!-- <li><a href="#">Home</a></li> -->
+                        <li><a class="active" href="/about.html">About</a></li>
+                        <li><a href="#">Program</a></li>
+                        <li><a href="#">Sponsor</a></li>
+                        <li><a href="#">News</a></li>
+                        <li><a href="#">CC Campaign</a></li>
+                    </ul>
+                </div>
+
+            </div>
+        </nav>
+    </header>
```

Comment:
- [ ] Kindly note that the header is divided into a top dark bar and a bottom white bar, the top short bar section with social media links, logout, and language links aligned to the right while taller bottom bar with a logo and a list of page links which is in place already. Kindy fix those that are not there.
## Your design
![image](https://user-images.githubusercontent.com/86200356/233099368-bd21a270-8019-4b1b-b3ec-80df639a7f10.png)

## Behance design
![image](https://user-images.githubusercontent.com/86200356/233099574-957f4374-226c-43f1-9439-94ab2a707b2b.png)